tgui/src/interfaces/suit_storage.ract

```diff
@@ -0,0 +1,71 @@
+{{#if data.isBroken}}
+  <ui-notice>
+    <span>Unit chamber is too contaminated to continue usage. Please call for a qualified individual to perform maintenance.</span> 
+  </ui-notice>
+{{/if}}
+
+{{#if data.isBaking}}
+  <ui-notice>
+    <span>Unit is cauterising contents with selected UV ray intensity. Please wait.</span>
+  </ui-notice>
+{{/if}}
+
+{{#if data.occupied}}
+  <ui-notice>
+    <span>WARNING: Biological entity detected inside the Unit's storage. Please remove.</span>
+  </ui-notice>
+{{/if}}
+
+{{#if !data.isBaking && !data.isBroken}}
+  <ui-display title="Storage" button>
+  {{#partial button}}
+      <ui-section>
+      {{#if !data.isLocked}}
+        <ui-button action='toggle_open' icon='{{data.isOpen ? "sign-out" : "sign-in"}}'>{{data.isOpen ? 'Close' : 'Open'}}</ui-button>
+      {{/if}}
+      {{#if !data.isOpen}}
+        <ui-button action='toggle_lock' icon='{{data.isLocked ? "unlock" : "lock"}}'>{{data.isLocked ? 'Unlock' : 'Lock'}}</ui-button>
+      {{/if}}
+      </ui-section>
+  {{/partial}}
+  {{#if data.isLocked}}
+    <ui-notice>
+      <span>LOCKED</span>
+    </ui-notice>
+  {{else}}
+      {{#if data.maintenance}}
+          <ui-notice>
+            <span>The panel is ridden with controls{{','}} buttons{{','}} and meters{{','}} labeled in strange signs and symbols that you cannot understand; probably the manufacturing world's language. Among other things, a few controls catch your eye...</span>
```

Comment:
This is fluff text and does not belong in a UI. Ideally, remove this in favor of a wire datum.